taccsite_cms/static/site_cms/css/src/_imports/trumps/s-breadcrumbs.css

```diff
@@ -0,0 +1,62 @@
+/*
+Breadcrumbs
+
+A list of links and to ancestor navigation (pages or sections that are a parent, grandparent, et cetera of the current page or section).
+
+Notice: A scope class is used to enforce appropriate semantic HTML.
+
+Markup:
+<!-- major navigation -->
+<nav class="s-breadcrumbs">
+  <ol>
+      <li><a href="/"><span>Home</span></a></li>
+      <li><a href="/"><span>Science</span></a></li>
+      <li><span>Publications</span></li>
+  </ol>
+</nav>
+<!-- minor navigation -->
+<ol class="s-breadcrumbs">
+    <li><a href="/"><span>Home</span></a></li>
+    <li><a href="/"><span>Science</span></a></li>
+    <li><span>Publications</span></li>
+</ol>
+
+Styleguide Trumps.Scopes.Breadcrumbs
+*/
+@import url("_imports/tools/x-truncate.css");
+
+
+
+
+
+/* Base i.e. Container */
+
+nav.s-breadcrumbs {
+  padding-top: 15px;
+
+  font-size: var(--global-font-size--x-small);
+
+  text-transform: uppercase;
+}
+
+
+
+/* Elements i.e. Content */
+
+nav.s-breadcrumbs ol {
+  list-style: none;
+}
```

Comment:
__Why not for `ol.s-breadcrumbs`?__ Because, a minor breadcrumb styling may use `list-style`. I do not want to make too many assumptions. I only have a design for major breadcrumb navigation.

Reply:
```suggestion
# FAQ: Unless/Untill designed otherwise, minor breadcrumbs may have default `list-style`
nav.s-breadcrumbs ol {
  list-style: none;
}
```